GSA/made-in-america
# [000] : remove content, compile app

## Description

removes description under waiver heading

## How to test

navigate [here](https://federalist-c275c827-2779-42ff-ac19-0331ebcacca9.app.cloud.gov/preview/gsa/made-in-america/hotfix/compile-app/waivers/), ensure expected text is not displayed.

## Additional information

needed to have app recompiled

To prepare for code review, please follow the following checks.
